presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/DockerContainer.java

```diff
@@ -168,6 +169,7 @@ protected void containerIsStarting(InspectContainerResponse containerInfo)
     @Override
     protected void containerIsStarted(InspectContainerResponse containerInfo)
     {
+        checkState(startupTime.isRunning(), "Container was not in starting state");
```

Comment:
make message `Container not started` and reference container name.

Reply:
Maybe we could just reset state instead? DockerContainer is reused when trying to start environment multiple times